### PR TITLE
introduce a pull request template

### PR DESCRIPTION
**Pull request by coorasse**

### CODE REVIEW

File: templates/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+### Description
+
+What I did (if it needs further explanation) and screenshots (if necessary).
```
Comment (schmijos) [inline templates/PULL_REQUEST_TEMPLATE.md]: Shorten to "What I did and maybe screenshots of it"

File: templates/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+### Description
+
+What I did and maybe screenshots.
+
+### General
+
+- [ ] I reviewed my code.
+- [ ] This PR is relatable to a ticket number (via branch name, title or description).
+- [ ] I updated the ticket on Redmine.
+- [ ] *This PR is deployed on the testing environment*
```
Comment (schmijos) [inline templates/PULL_REQUEST_TEMPLATE.md]: I'd leave this point away.

Comment (coorasse) [reply templates/PULL_REQUEST_TEMPLATE.md]: Why? I think it's useful to know it has been deployed on testing for the reviewer.

Comment (schmijos) [reply templates/PULL_REQUEST_TEMPLATE.md]: It's just not the normal case.

**Comment by coorasse** [comment]
I'll wait for @rbuetzer as well to have an opinion also from the other team. We discussed that already in our team but maybe there are inputs also from them.

File: templates/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (macav) [reply templates/PULL_REQUEST_TEMPLATE.md]: i'd say it can stay as optional point which i leave unticked if i didn't deploy it

**Pull request merged**